### ISSUE TITLE
Clone workflow template UI does not report length exceeded error.

### ISSUE THREAD

**Issue by aleksandrmelnikov** [onepanelio/core#458]
Not sure if this has been resolved.

I was cloning a Workflow Template with the name: `TensorFlow Training-clonevisibility`

Got this error:
```text
2020-07-30T20:26:46.335969856Z time="2020-07-30T20:26:46Z" level=error msg="Could not create workflow template." Error="Length of string exceeds 30" Namespace=default WorkflowTemplate="&{0 0001-01-01 00:00:00 +0000 UTC <nil>   TensorFlow Training-clonevisibility entrypoint: main\narguments:\n    parameters:\n    - name: source\n      value: https://github.com/onepanelio/tensorflow-examples.git\n    - name: command\n      value: \"python mnist/main.py --epochs=5\"\nvolumeClaimTemplates:\n  - metadata:\n      name: data\n    spec:\n      accessModes: [ \"ReadWriteOnce\" ]\n      resources:\n        requests:\n          storage: 2Gi\n  - metadata:\n      name: output\n    spec:\n      accessModes: [ \"ReadWriteOnce\" ]\n      resources:\n        requests:\n          storage: 2Gi\ntemplates:\n  - name: main\n    dag:\n      tasks:\n      - name: train-model\n        template: pytorch\n# Uncomment section below to send metrics to Slack\n#      - name: notify-in-slack\n#        dependencies: [train-model]\n#        template: slack-notify-success\n#        arguments:\n#          parameters:\n#          - name: status\n#            value: \"{{tasks.train-model.status}}\"\n#          artifacts:\n#          - name: metrics\n#            from: \"{{tasks.train-model.outputs.artifacts.sys-metrics}}\"\n  - name: pytorch\n    inputs:\n      artifacts:\n      - name: src\n        path: /mnt/src\n        git:\n          repo: \"{{workflow.parameters.source}}\"\n    outputs:\n      artifacts:\n      - name: model\n        path: /mnt/output\n        optional: true\n        archive:\n          none: {}\n    container:\n      image: tensorflow/tensorflow:latest\n      command: [sh,-c]\n      args: [\"{{workflow.parameters.command}}\"]\n      workingDir: /mnt/src\n      volumeMounts:\n      - name: data\n        mountPath: /mnt/data\n      - name: output\n        mountPath: /mnt/output\n  - name: slack-notify-success\n    container:\n      image: technosophos/slack-notify\n      command: [sh,-c]\n      args: ['SLACK_USERNAME=Worker SLACK_TITLE=\"{{workflow.name}} {{inputs.parameters.status}}\" SLACK_ICON=https://www.gravatar.com/avatar/5c4478592fe00878f62f0027be59c1bd SLACK_MESSAGE=$(cat /tmp/metrics.json)} ./slack-notify']\n    inputs:\n      parameters:\n      - name: status\n      artifacts:\n      - name: metrics\n        path: /tmp/metrics.json\n        optional: true\n 0 0 false false false nil map[accurate:yes special:no] <nil> <nil> 0 <nil> <nil> []}"
```

UI shows this:
![image](https://user-images.githubusercontent.com/10407733/88971288-98c99180-d268-11ea-8526-81d7737aee90.png)

**Comment by Vafilor** [comment]
@aleksandrmelnikov This should be fixed with the latest dev. Let me know if you are still seeing an issue.

**Comment by aleksandrmelnikov** [comment]
@Vafilor Verified this is fixed. Closed PR and added Milestone 0.13.0. Feel free to change.
